### ISSUE TITLE
/var/run/apel is not persistent under CentOS 7

### ISSUE THREAD

**Issue by dvandok** [apel/ssm#161]
On CentOS 7 with systemd, /var/run is a symlink to /run mounted on tmpfs. It starts out empty on reboot, so it is up to the software to populate it as necessary.
The common setting for the PID file under /var/run/apel results in failure on startup of the service because the pid file cannot be created.
The common way to deal with this seems to be to create the missing directories on the fly.

The latest rpm release https://github.com/apel/ssm/tree/3.2.0-1  still 'owns' /var/run/apel.

**Comment by drebatto** [comment]
Yes, it would be nice to have a configuration file for `systemd-tmpfiles`, e.g. `/usr/lib/tmpfiles.d/apel.conf`, with a single line:
```
d /var/run/apel 755 root root -
```
